generator/src/main/java/com/scottlogic/deg/generator/commandline/GenerateCommandLine.java

```diff
@@ -103,16 +103,23 @@
     private Boolean visualiseReductions = false;
 
     @CommandLine.Option(
-        names = {"-o"},
-        description = "Output format",
+        names = {"-f"},
```

Comment:
Repurposing `-o` is confusing (not to mention a breaking change).

Why not leave this as `-o` (matches [osql](https://docs.microsoft.com/en-us/sql/tools/osql-utility) - except for its case). Any either use an optional switch for the filename or something like a `-f` marker